data.json

```diff
@@ -2960,6 +2960,20 @@
             ],
             "furl": "https://www.troyhunt.com/rss/",
             "hnp": 13592
-        }
+        },
+        {
+            "name": "VisakanV'S Blog",
+            "url": "http://www.visakanv.com/blog/start-here/",
+            "tags": [
+                "culture",
+                "theory",
+                "strategy",
+                "reading",
+                "writing",
+                "people",
+                "travel"
+            ],
+            "furl": "http://www.visakanv.com/blog/rss/"
+        },
```

Comment:
thanks for the contribution, could you remove "," from at the end.

Reply:
```suggestion
        }
```